src/utils/getPlaceholder.js

```diff
@@ -0,0 +1,24 @@
+import {
+  isStreamNarrow,
+  isTopicNarrow,
+  isPrivateNarrow,
+  isGroupNarrow,
+} from '../utils/narrow';
+
+export const getInputPlaceholderFromNarrow = (narrow, auth, users) => {
```

Comment:
1. I agree. Call the file name itself `getInputPlaceholderFromNarrow`.
2. You can move the file to the `/compose` folder, as it will not be used from any other part of the code.
3. I suggested the name, sometimes longer name is more clear. In that case it is ok, but longer than that is too much :)